client/src/components/PostInfo.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { getText } from "../utils";
+
+const PostInfo = ({ post }) => {
+  const { id, title, img, content, author, created_at } = post;
+  console.log(post);
```

Comment:
Using console.log() is great for debugging but best left out of production for cleaner code.